test/suites/clustering.sh

```diff
@@ -280,6 +280,11 @@ test_clustering_containers() {
   apply_template2=$(LXD_DIR="${LXD_TWO_DIR}" lxc config get egg volatile.apply_template)
   [ "${apply_template1}" =  "${apply_template2}" ] || false
 
+  # Create backup and attempt to move container. Move should fail and container should remain on node3.
+  LXD_DIR="${LXD_THREE_DIR}" lxc query -X POST --wait -d '{\"name\":\"foo\"}' /1.0/instances/egg/backups
+  LXD_DIR="${LXD_THREE_DIR}" lxc move egg --target node1 | grep -q "Instance has backups"
```

Comment:
Should be change to:
```
! LXD_DIR="${LXD_THREE_DIR}" lxc move egg --target node1 || false
```

So it doesn't trip the shell when exiting non-zero.

Reply:
That makes sense. Fixed.